src/test/emulators/functionsEmulatorRuntime.spec.ts

```diff
@@ -681,7 +681,7 @@ describe("FunctionsEmulator-Runtime", () => {
 
         const data = await callHTTPSFunction(worker, frb);
         expect(JSON.parse(data)).to.deep.equal({ offset: 0 });
-      });
+      }).timeout(TIMEOUT_MED);
```

Comment:
I might be misunderstanding exactly how mocha's timeout works, but isn't this redundant with the .timeout() on the  next line that covers that whole describe block?

Reply:
You'd think, but nope.